ynr/apps/elections/uk/templates/candidates/_candidates_for_post.html

```diff
@@ -61,17 +61,31 @@
   {# Use the post label here, since multiple posts may be used on a page #}
   {% url 'constituency' election=election post_id=post_data.id ignored_slug=post_data.label|slugify as post_url %}
   {% url 'constituencies' election=election_data.slug as election_posts_url %}
-
   {% if user_can_lock %}
     <form method="post" action="{% url 'constituency-lock' election=election %}">
       {% csrf_token %}
       {% for field in lock_form %}{{ field }}{% endfor %}
-      <input type="submit" class="button small" value="{% if candidates_locked %}{% trans "Unlock candidate list" %}{% else %}{% trans "Lock candidate list" %}{% endif %}">
+      {% if current_user_suggested_lock %}
+        <p>
+          Locking disabled because you suggested locking this post.
+          Please ask someone else to review it as a double check.
```

Comment:
Happy with the logic, but a question about the messaging here:
Given there is no "request review" feature or messaging system or anything it seems like

> Please ask someone else to review it as a double check

is a CTA with no obvious path to fulfil it. i.e: As someone who has just suggested a lock here it sounds like I'm supposed to do something, but there's no obvious way to do it. Given users with appropriate permissions can see posts that need locking, would it be better to say something more like:

> Thanks for your contribution. An administrator will review the candidates for this area against the nomination paper

..or just not show anything except the absence of a 'suggest lock' button?

Reply:
Fair enough! I thought about linking to the Slackin but actually I think "Someone else will double check it soon." will do for now. Fixup pushed